pkgs/applications/networking/cluster/helmfile/default.nix

```diff
@@ -22,9 +22,14 @@ buildGoModule rec {
   nativeBuildInputs = [ installShellFiles ];
 
   postInstall = ''
+    mkdir completions
+    $out/bin/helmfile completion bash > completions/helmfile || true
```

Comment:
Can we instead do something like this, similar to other packages?

```bash
installShellCompletion --cmd helmfile \
  --bash <($out/bin/helmfile completion bash) \
  --fish <($out/bin/helmfile completion fish) \
  --zsh <($out/bin/helmfile completion zsh)
```

Reply:
Fixed, thanks for the suggestion!